recipe/activate-mdt.sh

```diff
@@ -1,20 +1,41 @@
-if [ -z "$MACOSX_DEPLOYMENT_TARGET" ]; then
-    export MACOSX_DEPLOYMENT_TARGET="@MACOSX_DEPLOYMENT_TARGET@"
-else
-    export CONDA_SYSROOT_@PLATFORM@_BACKUP_MACOSX_DEPLOYMENT_TARGET="${MACOSX_DEPLOYMENT_TARGET}"
+#!/usr/bin/env bash
+
+if [[ "${CONDA_BUILD:-0}" == "1" ]]; then
```

Comment:
Technically someone could install c_stdlib_osx-64 or c_stdlib_osx-arm64. I'd remove this condition.